index.html

```diff
@@ -438,6 +438,27 @@ <h2>Data Model</h2>
 <a>verification methods</a>.
       </p>
 
+      <p>
+All of the data model terms and types in this specification map to URLs. The
+vocabulary where these URLs are defined is the [[?SECURITY-VOCABULARY]]. The
+explicit mechanism that is used to perform this mapping in a secured document
+is the `@context` property.
+      </p>
+      <p>
+When processing a document as JSON-LD, the mapping mechanism is defined by
+JSON-LD [[JSON-LD11]]. When processing a document in a non-JSON-LD environment,
+the same mechanism is used to ensure that the terms have the same interpretation
+regardless of format, but with a simpler processing rule: Implementers
```

Comment:
Is this really a *simpler* rule?

`@context` directly maps JSON keys to URIs, end of rule.

The three-step described here seems much less simple, leaves a lot of handwaves (expected order of values? known good cryptographic hashes? known validity of contents?), and I rather expect it to be error prone, even when using the "static context files with a JSON Schema" method suggested later.

Reply:
I made an attempt at clarifying here: 9c556cb74083119d82644cee5cc76d8b4f025e0a

Reply:
Further revisions to the language were made in 0582d45a56c3c8aca4ee670c4abd59a2fd215ec4.